packages/insomnia-app/app/common/constants.js

```diff
@@ -257,6 +257,37 @@ const authTypesMap = {
   [AUTH_NETRC]: ['Netrc', 'Netrc File'],
 };
 
+// Sort Orders
+export const SORT_CUSTOM = 'custom';
+export const SORT_NAME_ASC = 'name-asc';
```

Comment:
You should be able to type this using a union - some duplication but it's safer to consume. See `GlobalActivity` for an example.

```suggestion
export type SortOption = 'custom' | 'name-asc' | ...;

export const SORT_CUSTOM: SortOption = 'custom';
export const SORT_NAME_ASC: SortOption = 'name-asc';
```

Later on, you can consume it like:
```js
export function getSortOrderName(sortOrder: SortOption): string {
	...
}
```

I selected `SortOption` arbitrarily, feel free to rename as you see fit.

Reply:
Cool. Refactored to a Union. Thanks for the reference.